docs/proposals/PROPOSAL_C.md

```diff
@@ -321,3 +267,38 @@ Gives response:
   "Chocolate Sprinkles Ice Cream"  
 }
 ```
+
+## Requirements
+
+### Filtering
+
+- Query for artifact: use `GET /v3/<name>/manifests/<reference>`.
+- Query for referrers to an artifact: use `GET /v3/<name>/referrers/<reference>`.
+- Query for referrers of a given type to an artifact: use `GET /v3/<name>/referrers/<reference>`, then use clinet to filter based on `description`. This is to allow artifact producers to include specific keywords rather than relying on IANA artifact types.
```

Comment:
```suggestion
- Query for referrers of a given type to an artifact: use `GET /v3/<name>/referrers/<reference>`, then use client to filter based on `description`. This is to allow artifact producers to include specific keywords rather than relying on IANA artifact types.
```